### PR TITLE
DTSSTCI-52

### PR DESCRIPTION
**Pull request by tomxelliott**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DTSSTCI-52

### Change description ###
Fix label content for Welsh button on landing page

Screenshot 2023-11-16 at 16 38 59

Screenshot 2023-11-16 at 16 38 56

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.23.2",
-    "@babel/eslint-parser": "^7.22.15",
+    "@babel/eslint-parser": "^7.11.0",
```
Comment (rajaram-nagarajan) [inline package.json:103]: Hope this package was downgraded by mistake.

Comment (tomxelliott) [reply package.json:103]: my yarn install must have downgraded it, will revert

File: yarn-audit-known-issues [hunk 2]
```diff
@@ -1 +1 @@
...
...commendation":"Upgrade to version 1.6.0 or later","cwe":["CWE-352"],"found_by":null,"deleted":null,"id":1094889,"references":"- https://nvd.nist.gov/vuln/detail/CVE-2023-45857\n- https://github.com/axios/axios/issues/6006\n- https://github.com/axios/axios/issues/6022\n- https://github.com/axios/axios/pull/6028\n- https://github.com/axios/axios/commit/96ee232bd3ee4de2e657333d4d2191cd389e14d0\n- https://github.com/axios/axios/releases/tag/v1.6.0\n- https://security.snyk.io/vuln/SNYK-JS-AXIOS-6032459\n- https://github.com/advisories/GHSA-wf5p-g6vw-rhxx","created":"2023-11-08T21:30:37.000Z","reported_by":null,"title":"Axios Cross-Site Request Forgery Vulnerability","npm_advisory_id":null,"overview":"An issue discovered in Axios 0.8.1 through 1.5.1 inadvertently reveals the confidential XSRF-TOKEN stored in cookies by including it in the HTTP header X-XSRF-TOKEN for every request made to any host allowing attackers to view sensitive information.","url":"https://github.com/advisories/GHSA-wf5p-g6vw-rhxx"}},"muted":[],"metadata":{"vulnerabilities":{"info":0,"low":0,"moderate":1,"high":0,"critical":0},"dependencies":456,"devDependencies":3,"optionalDependencies":0,"totalDependencies":459}}
```
Comment (rajaram-nagarajan) [inline yarn-audit-known-issues:1]: We need to create a separate JIRA ticket to fix this CVE if it is not introduced (I guess so) as part of this PR.

**Review by rajaram-nagarajan — DISMISSED** [review]
LGTM

File: yarn-audit-known-issues (hunk 2 above)
Comment (tomxelliott) [reply yarn-audit-known-issues:1]: the fix for it is in @gmmagruder PR: https://github.com/hmcts/sptribs-frontend/pull/524

**Review by rajaram-nagarajan — APPROVED** [review]

**Review by gmmagruder — APPROVED** [review]

**Pull request merged**